test/lib/linter/rules/snapshots/NoDeprecatedApi.ts.md

```diff
@@ -1908,6 +2102,14 @@ Generated by [AVA](https://avajs.dev).
             ruleId: 'no-globals',
             severity: 2,
           },
+          {
+            column: 3,
```

Comment:
This is a duplicate finding. The first one is created during XML analysis (but seems to have a wrong line/column info) and the new one comes from the transformed JS code.

Reply:
Thanks! Now the xml views & fragments are skipped in this check. They are being analyzed in the XML Parser